crates/astria-sequencer-types/src/sequencer_block_data.rs

```diff
@@ -514,4 +527,24 @@ mod test {
         assert!(matches!(last_commit_hash, Hash::Sha256(_)));
         assert!(expected_last_commit_hash.as_bytes() == last_commit_hash.as_bytes());
     }
+
+    #[test]
+    fn calculate_data_hash() {
+        use base64::{
+            engine::general_purpose::STANDARD,
+            Engine as _,
+        };
+
+        // data_hash is calculated from the txs in a block, where the leaves of the merkle tree are
```

Comment:
Please add the steps you took too generate this hash and which versions so that somebody else could generate this (or a similar) magic number

Reply:
added!